### DISCUSSION TITLE
unable to assemble Chloroplast genome for plant

### DISCUSSION THREAD

**Discussion by wx183** [Kinggerm/GetOrganelle#181] [Q&A]
I used GetOrganelle to assemble chloroplast genomes.I tried a lot of parameters, but I couldn't get the chloroplast genome together.I have attached two output log files for your kind perusal. Please let me know what should I do to improve this chloroplast sequence.
[H29 get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/9259792/H29.get_org.log.txt)
![H29](https://user-images.githubusercontent.com/110609383/182851816-7d8be925-f260-444d-bef3-cd55b275cf88.png)
[H32 get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/9259793/H32.get_org.log.txt)
![H32](https://user-images.githubusercontent.com/110609383/182851912-599f3a83-0e5e-44f9-b1f1-805256e50627.png)

**Comment by JianjunJin** [comment]
According to the log file and the `fastg` files provided elsewhere, these two samples may not have sufficient coverage (*check the coverage values in the log file) for some focal areas, potentially leading to the broken graph.

You may try to use the current result as the seed and smaller word size (`-w`) for another run; the success rate is low though.
